packages-kintone-customize/app-229-contracts/jsedit/api/getAllStoresMeeting.ts

```diff
@@ -4,7 +4,10 @@ import { ktRecord } from 'api-kintone';
 import { AppIds } from 'config';
 import { IStores } from 'types';
 
-
+/**
+ * 店舗の順番 ：会議資料用
+ * @returns 豊田、大林、美里、高浜・・・
+**/
```

Comment:
```suggestion
/**
 * kintoneアプリより"meetingNumber"毎にソートした店舗情報を取得します
 * @returns 店舗情報の配列
**/
```

修正案を記載しましたので、ご確認をお願いします。
なお、返り値は店舗レコードの配列になります。